site/docs/pages/checkout/checkout.mdx

```diff
@@ -121,6 +148,63 @@ export default function PayComponents() {
   </Checkout>
 </App>
 
+#### Using `chargeHandler`
+
+Alternatively, you can create charges dynamically using the Coinbase Commerce API [Create Charge](https://docs.cdp.coinbase.com/commerce-onchain/reference/creates-a-charge) endpoint and pass the chargeID into Checkout via the `chargeHandler` prop.
```

Comment:
nit: pass -> passing

or probably better:

Alternatively, you can create charges dynamically using the Coinbase Commerce API [Create Charge](https://docs.cdp.coinbase.com/commerce-onchain/reference/creates-a-charge) endpoint by passing the chargeID into Checkout via the `chargeHandler` prop.

or 

Alternatively, you can create charges dynamically by using the Coinbase Commerce API [Create Charge](https://docs.cdp.coinbase.com/commerce-onchain/reference/creates-a-charge) endpoint and passing the chargeID into Checkout via the `chargeHandler` prop.